tests/test_estimate_prob_two_rr_match.py

```diff
@@ -48,6 +49,15 @@ def test_prob_rr_match_dedupe(test_helpers, dialect):
         deterministic_rules, recall=0.9
     )
 
+    # test with BlockingRule object
+    deterministic_rules = [
```

Comment:
For clarity of blocking rules we do not want blocking rule library to be compatible with estimating lambda